deploy/role.yaml

```diff
@@ -174,6 +174,17 @@ rules:
   - watch
   - update
   - patch
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - prometheusrules
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - create
```

Comment:
Why create and update though? Are we going to create an automated remediation?

Reply:
Yes. See https://github.com/ComplianceAsCode/content/pull/7427